docs/examples/05-job_schedule.jl

```diff
@@ -0,0 +1,133 @@
+# This example demonstrates a job scheduling through adding the
+# number 100 to every component of the vector data. The master
+# assigns one element to each slave to compute the operation.
```

Comment:
```suggestion
# assigns one element to each worker to compute the operation.
```
and maybe change `master` to `root` as well (since that is MPI parlance)